set-java-home.bash

```diff
@@ -0,0 +1,14 @@
+asdf_update_java_home() {
+  local java_path
+  java_path="$(asdf which java)"
+  if [[ ! -z "${java_path}" ]]; then
+    export JAVA_HOME
+   JAVA_HOME="$(dirname "$(dirname "$(realpath "${java_path}")")")"
```

Comment:
Why all this complexity? I can have the correct path with `asdf where java`, right?

Reply:
I like the way you are thinking! Resisting complexity is very important.

I did implement initially using `asdf where java`, but I followed the recommendation from @thuandt so we could support the system java as well.

```
% asdf where java
System version is selected
% asdf which java
/usr/bin/java
```